### PR TITLE
add ubuntu 16.04 (#403)

### PR DESCRIPTION
**Pull request by RachelShemberg**

### CODE REVIEW

File: docs/release1907latest/deploymentguide/deploymentk8s.rst [hunk 1]
```diff
@@ -70,6 +70,7 @@ Deploy Rancher
 
 On the Linux **Rancher Server** machine, run Rancher::
 
+    mkdir /ericomshield
```
Comment (BenyH) [inline docs/release1907latest/deploymentguide/deploymentk8s.rst]: This command will creat the folder under root (which is not good)

mkdir ~/ericomshield

Comment (RachelShemberg) [reply docs/release1907latest/deploymentguide/deploymentk8s.rst]: ok

**Comment by RachelShemberg** [comment]
last commit is related to [#6880](https://github.com/EricomSoftwareLtd/SB/issues/6880)

File: docs/release1907latest/deploymentguide/deploymentk8s.rst [hunk 2]
```diff
@@ -54,9 +54,16 @@ Verify that Docker is installed properly::
 
     docker version
 
+All the machines should be in the same timezone. The recommendation is to configure the machines using NTP (Network Time Protocol).
+To do so, follow these steps. Run::
+
+	sudo yum install ntp
```
Comment (BenyH) [inline docs/release1907latest/deploymentguide/deploymentk8s.rst]: this is the command for Centos
please ask for Ubuntu

Comment (BenyH) [reply docs/release1907latest/deploymentguide/deploymentk8s.rst]: @RachelShemberg can you resolve this issue?

Comment (RachelShemberg) [reply docs/release1907latest/deploymentguide/deploymentk8s.rst]: as discussed, specific commands were replaced with a general comment

File: docs/release1907latest/deploymentguide/requirements.rst [hunk 3]
```diff
@@ -56,35 +56,39 @@ Ericom Shield requires these ports to be open on the network:
...
-    4789, UDP, Shield, Shield, "Inbound/outbound between any Shield servers, used for Cluster Management"
-    7946, TCP, Shield, Shield, "Inbound/outbound between any Shield servers, used for Cluster Management"
-    7946, UDP, Shield, Shield, "Inbound/outbound between any Shield servers, used for Cluster Management"
-    389, TCP, Shield - Authentication Proxy, LDAP Server, "Between Shield and LDAP server"
-    636, TCP, Shield - Authentication Proxy, LDAPS Server, "Between Shield and LDAP server"
-    88, TCP, Shield - Authentication Proxy, AD-Kerberos, "Required when using Kerberos authentication"
-    88, UDP, Shield - Authentication Proxy, AD-Kerberos, "Required when using Kerberos authentication"
-    80, TCP, Shield, Internet, ""
-    443, TCP, Shield, Internet, ""
+    3128, TCP, End users, Shield, "Inbound between End-Users and Shield"
```
Comment (BenyH) [inline docs/release1907latest/deploymentguide/requirements.rst]: End Users => Shield Proxy Servers

File: docs/release1907latest/deploymentguide/requirements.rst [hunk 4]
```diff
@@ -56,35 +56,39 @@ Ericom Shield requires these ports to be open on the network:
...
+    8443, TCP, "", Rancher, "Inbound and between all Shield servers. Used to connect to the Rancher Administration Console and between Shield servers to Rancher server"
+    30181, TCP, "", Admin Console, "Inbound. Used to connect to the Shield Administration Console"
```
Comment (BenyH) [inline docs/release1907latest/deploymentguide/requirements.rst]: Admins => Shield-Management

File: docs/release1907latest/deploymentguide/requirements.rst [hunk 5]
```diff
@@ -56,35 +56,39 @@ Ericom Shield requires these ports to be open on the network:
...
+    636, TCP, Shield Proxy, LDAPS Server, "Between Shield and LDAP server"
+    88, TCP, Shield Proxy, AD-Kerberos, "Required when using Kerberos authentication"
+    88, UDP, Shield Proxy, AD-Kerberos, "Required when using Kerberos authentication"
+    80, TCP, Shield, Internet, "Outbound internet connection"
+    443, TCP, Shield, Internet, "Outbound internet connection"
     53, TCP, Shield, DNS, ""
     53, UDP, Shield, DNS, ""
-    SSH 22, TCP, Shield, Shield, "Inbound/outbound between all Shield Nodes"
-    25, TCP, Browsers Farm, Internet, "Required when using SMTP for alerts and statistics"
+    25, TCP, Shield, SMTP server, "Required when using SMTP for alerts and statistics"
```
Comment (BenyH) [inline docs/release1907latest/deploymentguide/requirements.rst]: from Shield-Management?

Comment (RachelShemberg) [reply docs/release1907latest/deploymentguide/requirements.rst]: this needs to be opened from Shield management and from browser farm - to the smtp server

Comment (BenyH) [reply docs/release1907latest/deploymentguide/requirements.rst]: Please specify 
and "Resolve conversation"

File: docs/release1907latest/deploymentguide/requirements.rst (hunk 3 above)
Comment (RachelShemberg) [reply docs/release1907latest/deploymentguide/requirements.rst]: ok

File: docs/release1907latest/deploymentguide/requirements.rst (hunk 4 above)
Comment (RachelShemberg) [reply docs/release1907latest/deploymentguide/requirements.rst]: ok

**Pull request merged**